release.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+VERSION=`cat RELEASE`
+
+read -p "This will tag and create docker image for $VERSION. Continue (y/n)? " CONT
+
+if [ "$CONT" != "y" ]; then
+    echo "Aborting."
+    exit 0
+fi
+
+git commit -m "Updated version to $VERSION" RELEASE
+
+git push
+
+git tag "v$VERSION"
+
+git push origin "v$VERSION"
+
+pushd deployment/docker
+
+read -p "Build? " BUILDING
```

Comment:
What's the purpose of the read?  I don't see `$BUILDING` used anywhere.  Also, `build.sh` has it's own interactive check.

Reply:
As usual, your reviews are invaluable. I was trying to debug something. Will remove.